css/templates/e-commerce/page.css

```diff
@@ -202,16 +412,78 @@ body {
     background-color: var(--color-white);
     border: none;
     color: var(--color-primary);
-    font-size: var(--h5);
+    font-size: var(--h7);
   }
 
   .commerce-hero-content-theme-secondary,
   .commerce-hero-content-theme-secondary .commerce-hero-content-title {
-    color: var(--color-secondary);
+    color: var(--color-primary);
   }
 
   .commerce-hero-content-theme-secondary .ampstart-btn {
     color: var(--color-white);
-    background-color: var(--color-secondary);
+    background-color: var(--color-primary);
+  }
+}
+
+
+/* PRODUCT DETAILS */
+.icon-star,
+.icon-star-empty {
+  height: 14px;
+  width: 14px;
+  color: var(--color-font-accent);
+}
+
+.icon-star-empty {
+  color: #dadada;
```

Comment:
Should a new CSS var be introduced? This isn't the accent colour, should I use the same approach we did for social-color-icon?